<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>765</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Excel 2 LaTeX</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8"/>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>8</y>
      <width>851</width>
      <height>771</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QWidget" name="">
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Excel 2 LaTeX</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Einstellungen</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QGroupBox" name="groupBox_3">
               <property name="title">
                <string>Search</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QLineEdit" name="filePath"/>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton">
                    <property name="text">
                     <string>Browse</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QGroupBox" name="groupBox">
               <property name="title">
                <string>File</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="0" column="0">
                 <layout class="QGridLayout" name="gridLayout">
                  <item row="0" column="0">
                   <layout class="QHBoxLayout" name="horizontalLayout_2">
                    <item>
                     <widget class="QLabel" name="label_2">
                      <property name="font">
                       <font>
                        <pointsize>8</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>Delimiter (only csv):</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QComboBox" name="delimiter">
                      <item>
                       <property name="text">
                        <string>;</string>
                       </property>
                      </item>
                      <item>
                       <property name="text">
                        <string>,</string>
                       </property>
                      </item>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item row="1" column="0">
                   <widget class="QCheckBox" name="multipleSheets">
                    <property name="text">
                     <string>Multiple Sheets</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0">
                   <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                     <widget class="QLabel" name="label_3">
                      <property name="text">
                       <string>Sheetname :</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="sheetname">
                      <property name="enabled">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QGroupBox" name="groupBox_2">
               <property name="title">
                <string>Output</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_4">
                <item row="0" column="0">
                 <layout class="QGridLayout" name="gridLayout_3">
                  <item row="0" column="0">
                   <layout class="QHBoxLayout" name="horizontalLayout_6">
                    <item>
                     <widget class="QCheckBox" name="index">
                      <property name="text">
                       <string>Index</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QCheckBox" name="hlineAll">
                      <property name="text">
                       <string>hline(After each row)</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QCheckBox" name="hlineOutside">
                      <property name="text">
                       <string>hline(End and beginning)</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item row="1" column="0">
                   <layout class="QHBoxLayout" name="horizontalLayout_4">
                    <item>
                     <widget class="QLabel" name="label_5">
                      <property name="text">
                       <string>Endline :</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="endline">
                      <property name="text">
                       <string/>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item row="2" column="0">
                   <layout class="QHBoxLayout" name="horizontalLayout_7">
                    <item>
                     <widget class="QLabel" name="label_6">
                      <property name="text">
                       <string>Round by:</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QComboBox" name="roundBy">
                      <item>
                       <property name="text">
                        <string>Dont round</string>
                       </property>
                      </item>
                      <item>
                       <property name="text">
                        <string>0</string>
                       </property>
                      </item>
                      <item>
                       <property name="text">
                        <string>1</string>
                       </property>
                      </item>
                      <item>
                       <property name="text">
                        <string>2</string>
                       </property>
                      </item>
                      <item>
                       <property name="text">
                        <string>3</string>
                       </property>
                      </item>
                      <item>
                       <property name="text">
                        <string>4</string>
                       </property>
                      </item>
                      <item>
                       <property name="text">
                        <string>5</string>
                       </property>
                      </item>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Create</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>LaTeX Code:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QTextBrowser" name="debugTextBrowser"/>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAlign_Left">
   <property name="text">
    <string>Align Left</string>
   </property>
   <property name="toolTip">
    <string>Align Left</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>browseSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>805</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>837</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>createLaTeX()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>416</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>839</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filePath</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>returnPressedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>552</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>837</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>multipleSheets</sender>
   <signal>clicked(bool)</signal>
   <receiver>sheetname</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>463</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>610</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>multipleSheets</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>multiSheetSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>842</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>browseSlot()</slot>
  <slot>createLaTeX()</slot>
  <slot>returnPressedSlot()</slot>
  <slot>multiSheetSlot()</slot>
  <slot>indexSlot()</slot>
 </slots>
</ui>
